frontend/src/theme/textStyles.tsx

```diff
@@ -0,0 +1,29 @@
+const textStyles = {
+  "body-medium": {
+    fontFamily: "DM Sans",
```

Comment:
For the way we've done it, all I can do is make a variable `font = "DM Sans"`. There's another way we could've gone about the design system (see "Typography" section of [this link](https://chakra-ui.com/docs/styled-system/theme)), but I think that system wouldn't work as well as what we have now because the sizes and titles aren't consistent (example: "button-medium" is 18px while "body-medium" is 16px).

Reply:
I'm happy with having a var font